### DISCUSSION TITLE
AWS Hubs Cloud stack for use in mainland China

### DISCUSSION THREAD

**Discussion by mohrtw** [mozilla/hubs#3034] [Hubs Cloud Questions] [answered]
Has anyone had success or have attempted to get an AWS Hubs Cloud stack working for use in China? Or more generally getting a domain registered outside of China or a server outside of China working in China?

**Comment by robinkwilson** [comment, marked as answer]
From users, I've heard of HC deployments in the AWS region Tokyo work okay. I hope someone with personal experience can add to this!

On the Hubs Cloud side, the AWS regions in China are missing a component for HC to be deployed inside. And as of now, Unfortunately there are no plans for getting HC deployable using a service provider inside China.
